tedana/config.py

```diff
@@ -0,0 +1,6 @@
+"""Setting default values for ICA decomposition."""
```

Comment:
I'm noticing that the config file variables get to be used in both the Argument Parser and the variable defaults for `tedana_workflow` Given we've had a couple of bugs where those diverged, having them defined in one central place might be useful. I don't have a strong opinion on whether a `config.py` or global variables at the top of `tedana.py` make more sense.

Reply:
Without necessarily pushing for one solution or the other code-structure-wise, I would say that perhaps if this kind of solution is used, perhaps `defaults.py` would be better than `config.py`?

Also, while removing defaults from function arguments would reduce the problem, it might not remove it completely. Sometimes for a command-line option, you want to be able to state what the default behaviour or value will be within the help string. While it's easy to cross-reference within the `add_argument()` call what's quoted in the help string vs. what's set as the default, having both read from the same symbol is IMO more robust.